CautionaryAlertsApi/V1/UseCase/Interfaces/IGetCautionaryAlertByAlertIdUseCase.cs.cs

```diff
@@ -0,0 +1,12 @@
+using Hackney.Shared.CautionaryAlerts.Boundary.Response;
+using System.Threading.Tasks;
+using System;
+using Hackney.Shared.CautionaryAlerts.Infrastructure.GoogleSheets;
+
+namespace CautionaryAlertsApi.V1.UseCase.Interfaces
+{
+    public interface IGetCautionaryAlertByAlertIdUseCase
+    {
+        CautionaryAlertResponse ExecuteAsync(Guid personId, Guid alertId);
```

Comment:
Use case should not be returning Response objects - it violates the layer separation within the clean architecture.

Reply:
Use case should only deal with Domain objects.